OpenRA.Mods.Cnc/Graphics/VoxelLoader.cs

```diff
@@ -179,7 +179,7 @@ public ModelRenderData GenerateRenderData(VxlLimb l)
 			{
 				// Sheet overflow - allocate a new sheet and try once more
 				Log.Write("debug", "Voxel sheet overflow! Generating new sheet");
-				sheetBuilder.Current.ReleaseBuffer();
+				sheetBuilder.Current?.ReleaseBuffer();
```

Comment:
If sb.Current can be null here, it is not save to pass it to ModelRenderData below.

Reply:
this could be null only because this is in a catch of a try catch. It would probably be indicative of some other error if the sb.Current was null bellow

Reply:
Why should it be suddenly null inside the catch? Either we remove the `?.` if we know it cannot be null or we properly handle that below.